doc/association-rules.rst

```diff
@@ -0,0 +1,83 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+#### Inputs
+
+-  **Data**
+
+   Data set
+
+#### Outputs
```

Comment:
Remember how I was convinced this is a markdown document? It turns out it still isn't. This is not a heading format in reST. Please underline Inputs and Outputs with `~~~~~~~`, `^^^^^^^`, or similar (any new style of underline introduces the next level of heading). Sorry. :flushed: